packages/teleterm/src/main.ts

```diff
@@ -55,6 +60,16 @@ app.whenReady().then(() => {
     });
   }
 
+  const isWin = process.platform === 'win32';
+
+  const userDataPath = fs.realpathSync(app.getPath('userData'));
+  const installPath = fs.realpathSync(app.getAppPath());
```

Comment:
Don't the `app.getPath`/`app.getAppPath` already return full paths?

Reply:
In testing, yes. 
The docs say `Returns string - The current application directory.`.  so I wasn't 100% sure it would always return so a bit cautious here.

Reply:
What do you think about getting rid of it and addressing it if it ever becomes a problem?